VehicleUI/src/main/java/com/drivequant/drivekit/vehicle/ui/vehicledetail/fragment/VehicleDetailFragment.kt

```diff
@@ -212,6 +246,19 @@ class VehicleDetailFragment : Fragment() {
         DriveKitUI.analyticsListener?.trackScreen(getString(R.string.dk_tag_vehicles_detail), javaClass.simpleName)
     }
 
+    private fun updateVehicleImage() {
+        val customImage = viewModel.vehicle?.vehicleId?.let { VehicleCustomImageHelper.getImageUri(it) } ?: run { null }
```

Comment:
```suggestion
        val customImage = viewModel.vehicle?.vehicleId?.let { VehicleCustomImageHelper.getImageUri(it) }
```
Dans tous les cas on aura null en cas "d'erreur"